scribus/ui/fontcombo.cpp

```diff
@@ -320,3 +322,101 @@ void FontComboH::RebuildList(ScribusDoc *currentDoc, bool forAnnotation, bool fo
 	fontFamily->blockSignals(familySigBlocked);
 	fontStyle->blockSignals(styleSigBlocked);
 }
+
+fontFamilyDelegate::fontFamilyDelegate(QObject *parent):
+	QAbstractItemDelegate(parent)
+{
+}
+
+void fontFamilyDelegate::paint(QPainter *painter,
+							   const QStyleOptionViewItem &option,
+							   const QModelIndex &index) const
+{
+	QString text = index.data(Qt::DisplayRole).toString();
+	QFont font;
+	QStringList families = QFontDatabase().families();
+	QString name, style;
+	for (int i =0; i < families.size(); ++i)
```

Comment:
we gonna to need QFontDatabase for two reason:
1- AFAIK, there is no way to use freetype font face into Qpainter. Please if you know, guide me how.
2- QFontDatabase provide samples for all writing languages and It already initialized in Scribus.

The loop required to convert scribus font name in prefsManager to the correct Qfont name otherwise QtFont will fallback to the default system font.